### ISSUE TITLE
Delimiter option persisted among different runs

### ISSUE THREAD

**Issue by amercader** [frictionlessdata/goodtables-py#228]
This is likely and issue in tabulator, but flagging here as it affects `validate()`:

```python
In [1]: print open('testfile1.csv', 'r').read()
a;b;c;d
# comment
1;2;3;4

In [2]: print open('testfile2.csv', 'r').read()
a,b,c,d
1,2,3

In [3]: from goodtables import validate

In [4]: report = validate('testfile1.csv')

In [5]: (report['valid'], report['tables'][0]['headers'])
Out[5]: (True, [u'a;b;c;d'])

In [6]: report = validate('testfile2.csv')

In [7]: (report['valid'], report['tables'][0]['headers'])
Out[7]: (False, [u'a', u'b', u'c', u'd'])

In [8]: report = validate('testfile1.csv', delimiter=';', skip_rows=['#'])

In [9]: (report['valid'], report['tables'][0]['headers'])
Out[9]: (True, [u'a', u'b', u'c', u'd'])

In [10]: report = validate('testfile2.csv')

In [11]: (report['valid'], report['tables'][0]['headers'])
Out[11]: (True, [u'a,b,c,d'])      # <----- NOPE

```

The weird thing is that this only happens when you pass `delimiter` + another option (like `skip_rows`). If I only pass `delimiter` (see changes in `testfile1.csv`):

```python
In [1]: print open('testfile1.csv', 'r').read()
a;b;c;d
1;2;3;4


In [2]: print open('testfile2.csv', 'r').read()
a,b,c,d
1,2,3


In [3]: from goodtables import validate

In [4]: report = validate('testfile1.csv')

In [5]: (report['valid'], report['tables'][0]['headers'])
Out[5]: (True, [u'a', u'b', u'c', u'd'])

In [6]: report = validate('testfile2.csv')

In [7]: (report['valid'], report['tables'][0]['headers'])
Out[7]: (False, [u'a', u'b', u'c', u'd'])

In [8]: report = validate('testfile1.csv', delimiter=';')

In [9]: (report['valid'], report['tables'][0]['headers'])
Out[9]: (True, [u'a', u'b', u'c', u'd'])

In [10]: report = validate('testfile2.csv')

In [11]: (report['valid'], report['tables'][0]['headers'])
Out[11]: (False, [u'a', u'b', u'c', u'd'])    # <------ This is OK
```

**Comment by roll** [comment]
@amercader
Fixed in `tabulator@1.11.1` - please take a look

**Comment by amercader** [comment]
thanks @roll!
